### ISSUE TITLE
Error importing uPlot via webpack

### ISSUE THREAD

**Issue by mcrumm** [leeoniya/uPlot#169]
Hi @leeoniya! I ran into a problem importing the latest release into a webpack build.  It's actually the same problem I had when trying to use the GitHub repo as I started on phoenixframework/phoenix_live_dashboard#40 - at the time, I assumed it was a pre-release issue.

Importing uPlot v1.0.4:

```js
import uPlot from 'uplot'
```

Results in the following error in the web console:

```
Uncaught ReferenceError: Cannot access 'Dt' before initialization
```

at the following (obfuscated) line:

```js
Dt && G("mouseenter", r, t=>{
```

Earlier I built uPlot locally from 84840f6, and it imported and worked just find (though the same commit _failed_ using the GitHub repo and commit ref in npm, with the same error as above).

Strangely, the diff looks to be just the most recent commits before the new release:

```diff
diff --git a/assets/node_modules/uplot/dist/uPlot.esm.js b/assets/js/metrics_live/uPlot.esm.js
index d5a3a47..c453dd8 100644
--- a/assets/node_modules/uplot/dist/uPlot.esm.js
+++ b/assets/js/metrics_live/uPlot.esm.js
@@ -4,7 +4,7 @@
 *
~
 * uPlot.js (μPlot)
~
 * An exceptionally fast, tiny time series chart
~
 * https://github.com/leeoniya/uPlot 
-(v1.0.4)
+(v1.0.3)
~
 */
~
 
~
 function debounce(fn, time) {
~
@@ -1101,8 +1101,8 @@ function uPlot(opts, data, then) {
 
~
 	function delSeries(i) {
~
 		series.splice(i, 1);
~
 		legendRows.splice(i, 1)[0][0].parentNode.remove();
~
 		cursorPts.splice(i, 1)[0].remove();
~
 
~
 		// TODO: de-init no-longer-needed scales?
~
 	}
~
@@ -1160,7 +1160,7 @@ function uPlot(opts, data, then) {
 
~
 	let data0 = null;
~
 
~
 	function setData(_data, 
-_resetScales)
+_autoScaleX)
  {
~
 		self.data = _data;
~
 		data = _data.slice();
~
 		data0 = data[0];
~
@@ -1173,7 +1173,7 @@ function uPlot(opts, data, then) {
 
~
 		fire("setData");
~
 
~
 		
-_resetScales
+_autoScaleX
  !== false && autoScaleX();
~
 	}
~
 
~
 	self.setData = setData;
~
@@ -2104,7 +2104,7 @@ function uPlot(opts, data, then) {
 	function _alpha(i, value) {
~
 		series[i].alpha = value;
~
 
~
 		if 
-( legendRows)
+(legendRows)
~
 			legendRows[i][0].parentNode.style.opacity = value;
~
 	}
~
 
~

```

Let me know if I can help debug this further, or if it's just something I'm doing wrong - and thanks for all your great work on this library!

**Comment by leeoniya** [comment]
woops!

this was caused the by refactor that was needed for implementing the `addSeries` and `delSeries` APIs.

fixed in 1.0.5.

> and thanks for all your great work on this library!

thanks for using it!

**Comment by mcrumm** [comment]
@leeoniya 1.0.5 Works perfectly, thank you!!
